app/assets/stylesheets/application.css.scss

```diff
@@ -301,7 +302,7 @@ $daria-color-lt : #825fb9;
       border-color: $daria-color;
     }
   }
-  
+
```

Comment:
appreciate you leaving the code in better shape than you found it! Especially important for the CSS since we generally touch it to fix things, and cleaner code makes it easier to find the stuff to fix.